psga/sars_cov_2/ncov2019_artic.nf

```diff
@@ -4,14 +4,14 @@
  */
 process ncov2019_artic_nf_pipeline_illumina {
   publishDir "${params.output_path}/ncov2019-artic", mode: 'copy', overwrite: true, pattern: 'output_{bam,fasta,plots,typing,variants}/*'
+  publishDir "${params.output_path}", mode: 'copy', overwrite: true, pattern: 'reheadered_fasta/*.fasta'
```

Comment:
There is a ticket for porting the nested code below from bash to external python script. This is out of the scope for this ticket.